arxiv/taxonomy/category.py

```diff
@@ -0,0 +1,70 @@
+"""Provides a convenience class for working with arXiv categories."""
```

Comment:
Implements the logic that was on the browse Category domain class as an extension of the builtin ``str`` type.